packages/core/src/dialects/snowflake/query-generator-typescript.ts

```diff
@@ -34,15 +33,13 @@ export class SnowflakeQueryGeneratorTypeScript extends AbstractQueryGenerator {
         'createDatabaseQuery',
         this.dialect.name,
         CREATE_DATABASE_QUERY_SUPPORTABLE_OPTIONS,
-        CREATE_DATABASE_QUERY_SUPPORTED_OPTIONS,
+        new Set<keyof CreateDatabaseQueryOptions>([]),
```

Comment:
Just for consistency; can we keep the CREATE_DATABASE_QUERY_SUPPORTED_OPTIONS? Like we do for LIST_DATABASES_QUERY_SUPPORTED_OPTIONS

Reply:
make sense, reverted it